kratos/includes/kratos_parameters.h

```diff
@@ -307,6 +408,38 @@ class Parameters
 //         mpvalue->SetString(rapidjson::StringRef(value.c_str()));
 //        mpvalue->SetString(value.c_str(), value.length());
     }
+    void SetVector(const Vector& vec)
```

Comment:
Const reference looks ok to me. Why do you think it is wrong? Anyway you are copying it to the is In, at least by passing a reference you avoid copying it twice